### PR TITLE
Update for debian 10

### PR DESCRIPTION
**Pull request by Tontonjo**
Add commands for debian 10 buster and error loading nvidia smi driver inside container

### CODE REVIEW

**Comment by dkanada** [comment]
There seem to be linting issues reported by the CI we use.

**Comment by joshuaboniface** [comment]
Approved pending fix of linting errors:

```
general/administration/hardware-acceleration.md:193 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
general/administration/hardware-acceleration.md:263 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
general/administration/hardware-acceleration.md:265 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
general/administration/hardware-acceleration.md:265 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
general/administration/hardware-acceleration.md:267 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
general/administration/hardware-acceleration.md:268:4 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
general/administration/hardware-acceleration.md:269 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
general/administration/hardware-acceleration.md:269 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
general/administration/hardware-acceleration.md:271 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
```

File: general/administration/hardware-acceleration.md [hunk 1]
```diff
@@ -253,9 +259,17 @@ usermod -aG video user
 Once the container is started you can again validate access to the host ressources.
 
 ```sh
-docker exec -it jellyfin ldconfig -p | grep cuvid
-docker exec -it jellyfin ldconfig -p | grep libnvidia-encode.so.1
+docker exec -it jellyfin nvidia-smi
+```
```
Comment (joshuaboniface) [inline general/administration/hardware-acceleration.md]: E.G. here and the lines below.

File: general/administration/hardware-acceleration.md [hunk 2]
```diff
@@ -250,15 +253,21 @@ You may need to add this user to the video group.
...
-docker exec -it jellyfin ldconfig -p | grep cuvid
-docker exec -it jellyfin ldconfig -p | grep libnvidia-encode.so.1
+docker exec -it jellyfin nvidia-smi
 ```
 
-Now go in Jellyfin playback settings, enable Nvidia NVENC and select target codecs depending on what your GPU support
-try to play any file needing a transcode. Changing the bitrate is a good way to try this.
+If you get driver information, everything is fine but if you get an error like `couldn't find libnvidia-ml.so library in your system` you need to run the following command.
+
+```
```
Comment (dkanada) [inline general/administration/hardware-acceleration.md:264]:
```suggestion
```sh
```

**Pull request merged**